src/types/video.ts

```diff
@@ -223,6 +223,7 @@ export type AudioOutput = 'speaker' | 'earpiece';
 
 export type ControlsStyles = {
   hideSeekBar?: boolean;
+  hideDuration?: boolean;
```

Comment:
Please add the missing TypeScript to this file `src/specs/VideoNativeComponent.ts`
```
type ControlsStyles = Readonly<{
  hideSeekBar?: boolean;
  hideDuration?: boolean;
  seekIncrementMS?: Int32;
}>;
```